packages/fleet_server/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.3.1"
+  changes:
+    - description: Fix agent template to avoid server:null
```

we could have a more user friendly description

```suggestion
    - description: Fix agent template to avoid server:null that caused invalid policy 
```